source/grid/tria.cc

```diff
@@ -8995,28 +8995,79 @@ Triangulation (const Triangulation<dim, spacedim> &other)
 
 
 
+#ifdef DEAL_II_WITH_CXX11
+template <int dim, int spacedim>
+Triangulation<dim, spacedim>::
+Triangulation (Triangulation<dim, spacedim> &&tria)
+  :
+  Subscriptor(static_cast<Subscriptor&&>(tria)),
+  smooth_grid(tria.smooth_grid),
+  periodic_face_pairs_level_0(tria.periodic_face_pairs_level_0),
+  periodic_face_map(tria.periodic_face_map),
+  levels(tria.levels),
+  faces(tria.faces),
+  vertices(std::move(tria.vertices)),
+  vertices_used(tria.vertices_used),
```

Comment:
Sure. Thanks for experimenting. I would have expected that `rvalue_reference.member_variable` also ends up being an rvalue reference, so that this calls the move constructor of the member variable to be initialized, but I may of course be wrong about this. I often end up writing little 20-line programs to test this when I get confused.